services/user/CommonApi/common/packages/common-lib/src/supervisor.ts

```diff
@@ -216,3 +235,5 @@ export class Supervisor {
         iframe.contentWindow.postMessage(message, this.supervisorSrc);
     }
 }
+
+export const getSupervisor = Supervisor.getInstance;
```

Comment:
Any reason not to also have `export const supervisor = Supervisor.getInstance();`?

Reply:
Because then it eagerly instantiates the supervisor without any options. I tried multiple ways to make that work, but it snarls everything up. I imagine that with some more work on the bundling of common-lib, I could get it to work.

I could call the existing import `supervisor` instead of `getSupervisor` and then an app could just call `supervisor(...)` when ready if you prefer that.